command/monitor.go

```diff
@@ -0,0 +1,88 @@
+package command
+
+import (
+	"strings"
+
+	"github.com/jrasell/levant/levant"
+	"github.com/jrasell/levant/levant/structs"
+	"github.com/jrasell/levant/logging"
+)
+
+// MonitorCommand is the command implementation that allows users to monitor
+// a deploy until it completes
```

Comment:
is it not monitoring the job until it completes?

Reply:
correct.  will fix.